city_scrapers/spiders/wicks_sedgwick_bocc.py

```diff
@@ -0,0 +1,105 @@
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class WicksSedgwickBoccSpider(CityScrapersSpider):
+    name = "wicks_bocc"
```

Comment:
**nitpick:** Sorry, I know, easy to miss, the `name` class variable should also be updated to `wicks_sedgwick_bocc`. 😬